src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,136 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.22.0.dev2 (10/13/2019)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+* Deprecate mypy-version flag and add new version flag (#8463)
+  `Issue #7627 <https://github.com/pantsbuild/pants/issues/7627>`_
+  `PR #8463 <https://github.com/pantsbuild/pants/pull/8463>`_
+
+* Fix tensorflow==1.14 globbing with platform-specific native_lib_names (#8378)
+  `PR #8378 <https://github.com/pantsbuild/pants/pull/8378>`_
+
+* Error out if the user explicitly tries to test a non-test tgt. (#8402)
+  `PR #8402 <https://github.com/pantsbuild/pants/pull/8402>`_
+
+* Revert "Format our python code with Black (#8391)" (#8409)
+  `PR #8391 <https://github.com/pantsbuild/pants/pull/8391>`_
```

Comment:
Probably don’t want to link to this one. This was the original that got reverted

Reply:
This was in New Features for the last release. Happy to put elsewhere but I want to make sure you consider that piece before doing so just yet

Reply:
Oh hm, it should have gone in refactoring last release. The change only impacted the internal codebase. The new feature was a separate PR adding Black as a V2 rule. 

Not sure what to do here given the prior release..I’m fine either way but would bias towards putting this in tooling.

Reply:
will follow your instinct then